### PR TITLE
Add spanish language support

### PR DESCRIPTION
**Pull request by buttercubz**
> Always use `next` branch as destination branch for PRs, not `main`

### CODE REVIEW

File: lang/es.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "auth.confirm_password": "Confirmar contraseña",
+    "auth.forgot_password": "¿Olvidaste tu contraseña?",
+    "auth.forgot_password_send_email": "Enviar correo de recuperación de contraseña",
+    "auth.register_now": "Registrar",
+    "auth.logout": "Cerrar sesión",
+    "auth.register": "Registrar",
+    "auth.registration_disabled": "El registro está desactivado. Por favor contacta con el administrador.",
+    "auth.reset_password": "Cambiar contraseña",
+    "auth.failed": "Las credenciales no coinciden con nuestro registro..",
+    "auth.failed.callback": "Fallo el proceso de inicio de sesión con el proveedor.",
```
Comment (osauceda-ads) [inline lang/es.json:18]:
```json
   "auth.failed.callback": "Falló el proceso de inicio de sesión con el proveedor.",
```

File: lang/es.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "auth.failed.password": "La contraseña es incorrecta.",
+    "auth.failed.email": "No encontramos un usuario con ese correo.",
+    "auth.throttle": "Demasiados intentos. Por favor Inténtalo en :seconds segundos.",
```
Comment (osauceda-ads) [inline lang/es.json:21]:
```json
    "auth.throttle": "Demasiados intentos. Por favor inténtalo en :seconds segundos.",
```

File: lang/es.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "input.name": "Nombre",
+    "input.email": "Correo",
+    "input.password": "Contraseña",
+    "input.password.again": "Escribe la contraseña otra vez",
+    "input.code": "Código de único uso",
+    "input.recovery_code": "Código de recuperación",
+    "button.save": "Guardar",
+    "repository.url": "<span class='text-helper'>Examples</span><br>Para repositorios publicos, usar <span class='text-helper'>https://...</span>.<br>Para repositorios privados, usar <span class='text-helper'>git@...</span>.<br><br>https://github.com/coollabsio/coolify-examples <span class='text-helper'>main</span> la rama 'main' será seleccionada.<br>https://github.com/coollabsio/coolify-examples/tree/nodejs-fastify <span class='text-helper'>nodejs-fastify</span> la rama 'nodejs-fastify' será seleccionada.<br>https://gitea.com/sedlav/expressjs.git <span class='text-helper'>main</span> la rama 'main' será seleccionada.<br>https://gitlab.com/andrasbacsai/nodejs-example.git <span class='text-helper'>main</span> la rama 'main' será seleccionada."
```
Comment (osauceda-ads) [inline lang/es.json:29]:
```json
    "repository.url": "<span class='text-helper'>Examples</span><br>Para repositorios públicos, usar <span class='text-helper'>https://...</span>.<br>Para repositorios privados, usar <span class='text-helper'>git@...</span>.<br><br>https://github.com/coollabsio/coolify-examples <span class='text-helper'>main</span> la rama 'main' será seleccionada.<br>https://github.com/coollabsio/coolify-examples/tree/nodejs-fastify <span class='text-helper'>nodejs-fastify</span> la rama 'nodejs-fastify' será seleccionada.<br>https://gitea.com/sedlav/expressjs.git <span class='text-helper'>main</span> la rama 'main' será seleccionada.<br>https://gitlab.com/andrasbacsai/nodejs-example.git <span class='text-helper'>main</span> la rama 'main' será seleccionada."
```

**Comment by andrasbacsai** [comment]
I will add the suggested changes (@osauceda-ads). 

Thanks @buttercubz and @osauceda-ads.

**Pull request merged**